pulp_smash/utils.py

```diff
@@ -149,3 +149,25 @@ def setUpClass(cls):
         stack.callback(testcase.tearDownClass)
         yield stack
         stack.pop_all()
+
+
+def get_pulp_setting(cli_client, setting_name, python_type="str"):
+    """Retrieves the value of a Pulp setting from system under test."""
+    if cli_client.transport in ["docker", "podman", "kubectl"]:
+        manager = "pulpcore-manager"
+    else:
+        ps_output = cli_client.run(("ps", "ax")).stdout.splitlines()
+        for line in ps_output:
+            if "pulpcore" in line:
+                if "bin/python" in line.split()[4]:
+                    bin_dir = line.split()[4].rsplit("/", maxsplit=1)[0]
+        manager = "{}/pulpcore-manager".format(bin_dir)
+    command = "from django.conf import settings; print(repr(getattr(settings, '{}', None)))".format(
+        setting_name
+    )
+    value = (
+        cli_client.run((manager, "shell", "-c", command))
+        .stdout.rstrip("\n")
+        .replace("'", "")
+    )
+    return value
```

Comment:
```suggestion
    return python_type(value)
```

Reply:
I did not mean for python_type kwarg to remain. It was needed for the previous implementation with dynaconf.

Reply:
So this will always return a `repr` string of the config option.
Maybe using json to actually get the proper type would be an option?